Gruntfile.js

```diff
@@ -11,6 +11,6 @@ module.exports = function(grunt) {
 
   grunt.initConfig(configs);
 
-  grunt.registerTask('test', ['webpack:test', 'mochaTest', 'eslint']);
+  grunt.registerTask('test', ['jscs', 'webpack:test', 'mochaTest', 'eslint']);
```

Comment:
It's probably preferable to have jscs at the end alongside eslint because when developing tests are more important output, than it bailing on little stylistic issues.